ignore/src/types.rs

```diff
@@ -142,6 +142,7 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("pdf", &["*.pdf"]),
     ("php", &["*.php", "*.php3", "*.php4", "*.php5", "*.phtml"]),
     ("pod", &["*.pod"]),
+    ("powershell", &["*.ps1", "*.psd1", "*.psm1"]),
```

Comment:
> I also had a slight concern with potential confusion with postscript - one reason PowerShell doesn't use the extension ps.

Yeah, I thought about that too, but postscript could use `post` I think.

> This would be the second longest type, and there are many somewhat long types, e.g.:

Yeah, I haven't been terribly conscientious with vetting each name that's added. We do at least have `md` for `markdown`.

> At any rate, I posted a survey on Twitter, so we'll see what the feedback looks like.

Funny. I retweeted it. :-)